addons/mail/static/src/scss/many2one_avatar_user.scss

```diff
@@ -1,6 +0,0 @@
-.o_field_many2one_avatar {
-    .o_m2o_avatar > img:hover {
-        cursor: pointer;
-        filter: brightness(0.8);
-    }
-}
```

Comment:
This is gone?

Reply:
This file was not included in the manifest and as such, has no effect. As best as I can tell, its role is now covered by `m2x_avatar_user.scss`

Reply:
ok, good catch